android/src/main/java/com/thebluealliance/androidclient/accounts/AccountHelper.java

```diff
@@ -96,8 +97,17 @@ public static GoogleAccountCredential getSelectedAccountCredential(Context conte
         return credential;
     }
 
-    public static TbaMobile getTbaMobile(GoogleAccountCredential credential) {
+    public static TbaMobile getTbaMobile(Context context, GoogleAccountCredential credential) {
         TbaMobile.Builder tbaMobile = new TbaMobile.Builder(HTTP_TRANSPORT, JSON_FACTORY, credential);
+        if (Utilities.isDebuggable()) {
+            // If a debug build, set Cloud Endpoints to point to a dev TBA instance
+            String appspotId = Utilities.readLocalProperty(context, "appspot.projectId");
+            if (!appspotId.isEmpty()) {
```

Comment:
If the debug appspot ID is empty, will TbaMobile just point to the default site, or should we provide some sensible default for apiUrl?